randovania/games/prime1/json_data/Phazon Mines.txt

```diff
@@ -1257,7 +1257,8 @@ Extra - aabb: [122.126015, -44.231895, -102.18276, 201.19897, 105.61425, -30.731
   > Door to Quarantine Access A
       All of the following:
           # The Security Drone event triggers door locks on all doors in Central Dynamo except for the door to Quarantine Access A. However, if Samus leaves this room without picking up the item, the doors to Save Station Mines B and Dynamo Access will then permanently be sealed, making it impossible to backtrack through the room.
```

Comment:
I just now read this comment. Even though it's not your change, I don't think this is true at all. I'm almost certain you can still unlock the doors after passing through once.

Reply:
I now understand what you mean. Yeah. the doors do still unlock after the drone is defeated and the item collected.